File: Cs/Extensions.cs
using System;

namespace HalloJoe.Spin
{   
    public static class Extensions
    {
        /// <summary>
        /// Shorthand for generating a random spin result as string only
        /// </summary>
        /// <param name="s"></param>
        /// <param name="rnd"></param>
        /// <returns></returns>
        public static string Spin(this string s, Random rnd = null) => TextSpinner.Spin(s, rnd ?? new Random());
    }
}

File: Vb/Extensions.vb
Imports Microsoft.VisualBasic
Imports System.Runtime.CompilerServices
Imports HalloJoe.Spin.Vb
Namespace HalloJoe.Spin.Vb
    Module SpinExtensions
        ''' <summary>
        ''' Shorthand for generating a random spin result as string only
        ''' </summary>
        ''' <param name="s"></param>
        ''' <param name="rnd"></param>
        ''' <returns></returns>
        <Extension()>
        Public Function Spin(ByRef s As String, Optional ByVal rnd As Random = Nothing) As String
            If rnd Is Nothing Then rnd = New Random()
            Return Spin(s, rnd)
        End Function
    End Module
End Namespace
